src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -370,7 +370,8 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     void Appearances::CurrentColorScheme(const ColorScheme& val)
     {
-        Appearance().ColorSchemeName(val.Name());
+        Appearance().DarkColorSchemeName(val.Name());
```

Comment:
Wouldn't that be handled by

```c++
                else if (settingName == L"DarkColorSchemeName" || settingName == L"LightColorSchemeName")
                {
                    _PropertyChangedHandlers(*this, PropertyChangedEventArgs{ L"CurrentColorScheme" });
                }
```

up on L246?

Reply:
I don't believe so. You're going into the inner TSM Appearance object, which has no notification capabilities (?), and changing the datamodel field directly. Unless I am looking at the wrong layer, and Appearance here is an AppearanceViewModel? If so, disregard, but then know that this will trigger a spurious double-notification :)

Reply:
Are we sure? cause If I add 
```c++
_PropertyChangedHandlers(*this, PropertyChangedEventArgs{ L"CurrentColorScheme" });
```
that causes the app to crash when changing the scheme in the UI

Reply:
> I don't believe so. You're going into the inner TSM Appearance object, which has no notification capabilities (?), and changing the datamodel field directly. Unless I am looking at the wrong layer, and Appearance here is an AppearanceViewModel? If so, disregard, but then know that this will trigger a spurious double-notification :)

For completeness of this thread, Appearance was an AppearanceViewModel.